iceoryx_hoofs/include/iceoryx_hoofs/log/platform/logger.hpp

```diff
@@ -31,12 +31,17 @@ using pbb::logLevelFromEnvOr;
 using Logger = pbb::Logger<pbb::ConsoleLogger>;
 using TestingLoggerBase = pbb::Logger<pbb::ConsoleLogger>;
 
-/// @todo iox-#1345 make this a compile time option since if will reduce performance but some logger might want
-/// to do the filtering by themselves
+/// @todo iox-#1345 make this a compile time option
```

Comment:
It is already compile time, did you mean cmake argument or compiler define?

Reply:
Both. I clarified it